### DISCUSSION TITLE
Add "Export to PDF" capability to insights dashboards

### DISCUSSION THREAD

**Discussion by mape09** [directus/directus#12926] [Feature Requests]
Is it possible to generate a PDF with logo like the analyst part and put some data from the base ?
![Capture](https://user-images.githubusercontent.com/103728185/164440138-a3fb553d-5614-45c6-a86b-95d8632a79b9.PNG)

**Comment by rijkvanzanten** [comment]
There's no native "Export to PDF" functionality as of right now. I'll convert this to be the feature request for that 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
